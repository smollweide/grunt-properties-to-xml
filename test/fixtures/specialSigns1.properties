# base i18n properties
group2.item2 = Item 2
group1.item1 = Item 1
group1.item2 = Item 2
group2.item3 = Item 3 = something [{0}]
group1.item3 = Item 3 = something & something else
group2.item1 = Item " 1