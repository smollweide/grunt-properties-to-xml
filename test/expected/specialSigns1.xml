<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0"
	jcr:title="React"
	jcr:primaryType="nt:unstructured"
	React.group2.item2="{String}Item 2"
	React.group1.item1="{String}Item 1"
	React.group1.item2="{String}Item 2"
	React.group2.item3="{String}Item 3 = something \[{0}\]"
	React.group1.item3="{String}Item 3 = something &amp; something else"
	React.group2.item1="{String}Item &quot; 1"
/>